extension/ts/content/extensions/author/authorPage/index.ts

```diff
@@ -68,6 +77,14 @@ window.addEventListener("load", async () => {
 		const container = document.querySelector<HTMLTableCellElement>("table.authorContentTable td.middle");
 		if (container) {
 			appendUI(container, {onSync, onEdit, onSave, onPush, onPull, onCancel}, getTags);
+			const input = document.getElementById(AUTHOR_TAG_INPUT_ID) as HTMLInputElement;
+			console.log(input);
+			const {showTagValidator} = appendTagValidator(
+				(listener) => listeners.add(listener),
+				(listener) => listeners.delete(listener),
+				input
+			);
```

Comment:
i might be misunderstanding, but before we were using `onFormRender` and passing in the `onSave` and `offSave` handlers from that into `appendTagValidator`, but here we're not using `onFormRender` so we need to write our own listeners and have that variable around. why the different approaches?

Reply:
Just because we don't have a nice Author page API the same way we have a Book page API.

Ideally we'd have an `onAuthorFormRender` and a `onBookFormRender`

Reply:
New issue time